backend/src/Equinor.ProjectExecutionPortal.WebApi/Controllers/BundleController.cs

```diff
@@ -1,15 +1,16 @@
 ﻿using Equinor.ProjectExecutionPortal.FusionPortalApi.Apps;
+using Microsoft.AspNetCore.Authentication.JwtBearer;
 using Microsoft.AspNetCore.Authorization;
 using Microsoft.AspNetCore.Mvc;
 
 namespace Equinor.ProjectExecutionPortal.WebApi.Controllers
 {
-    //[Authorize(AuthenticationSchemes = JwtBearerDefaults.AuthenticationScheme)]
+    [Authorize(AuthenticationSchemes = JwtBearerDefaults.AuthenticationScheme)]
```

Comment:
Will this add authentication to the en point? then i think it will fail in the frontend.

Reply:
It will add bearer token auth, not cookie.